examples/SLAM/SLAM-camera-example.wlp

```diff
@@ -0,0 +1,685 @@
+{
+    "objects": {
```

Comment:
Our official example project filenames follow `CamelCase` naming and omit the "Example" suffix as they are usually in a folder called `examples` (like here also), i.e. this could be `SLAMCamera.wlp` (Make sure to also change "Project Settings > name" / `settings/project/name` in the JSON)